src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain;
+
+public class Car {
+    private static final int CAR_LENGTH = 5;
+    private String name;
```

Comment:
```suggestion
    private static final int CAR_LENGTH = 5;
    
    private String name;
```
`상수`와 `인스턴스 변수` 사이는 줄 바꿈을 통해 명확히 구분하는게 좋을 것 같아요